pkgs/servers/http/pomerium/external-envoy.diff

```diff
@@ -0,0 +1,50 @@
+diff --git a/pkg/envoy/envoy.go b/pkg/envoy/envoy.go
+index e32cfc29..4303892d 100644
+--- a/pkg/envoy/envoy.go
++++ b/pkg/envoy/envoy.go
+@@ -8,9 +8,9 @@ import (
+ 	"errors"
+ 	"fmt"
+ 	"io"
++	"io/fs"
+ 	"os"
+ 	"os/exec"
+-	"path"
+ 	"path/filepath"
+ 	"regexp"
+ 	"strconv"
+@@ -36,8 +36,14 @@ import (
+ 
+ const (
+ 	configFileName = "envoy-config.yaml"
++	workingDirectoryName = ".pomerium-envoy"
++	embeddedEnvoyPermissions     fs.FileMode = 0o700
++	embeddedDirectoryPermissions fs.FileMode = 0o755
```

Comment:
Is this patch custom made? If so please clean it up, eg embeddedDirectoryPermissions is not used. If not please fetchpatch it.

Reply:
Yes -- it's an adaptation of the functionality that was removed upstream that allowed use of external envoy -- removed unused variable.